subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/platforms/JavaPlatformResolveIntegrationTest.groovy

```diff
@@ -864,6 +864,82 @@ class JavaPlatformResolveIntegrationTest extends AbstractHttpDependencyResolutio
         //Shape of the graph is not checked as bug was failing resolution altogether
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/20684")
+    def "multiple platform deselection - reselection does not leave pending constraints in graph - different issue"() {
```

Comment:
Could not tell how this case was different from the one above. If the scenario name is not a good place to summarize that, maybe add a comment that would provide that context? (maybe a copy of the commit message would suffice? Or do we favor commit messages over inline comments?)

Reply:
I am not sure I would add such a detail here. More info is available on the commit comment indeed.